packages/drivers/replay-driver/src/replayDocumentServiceFactory.ts

```diff
@@ -39,8 +53,9 @@ export class ReplayDocumentServiceFactory implements IDocumentServiceFactory {
         resolvedUrl: IResolvedUrl,
         logger?: ITelemetryBaseLogger,
     ): Promise<IDocumentService> {
+        const replayLogger = logger && new ReplayDriverTelemetryLogger(logger);
```

Comment:
Note that you do not need to test for undefined when using ChildLogger.create() - it's happy with creating null logger on demand